apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -234,6 +258,40 @@ class LinuxDownloads extends React.Component {
   }
 }
 
+const FETCH_STATUS_STYLE = {
+  fontSize: 'large',
+  margin: '0.5em 0',
+};
+
+const FetchingLatestVersionMessage = () => (
+  <div style={FETCH_STATUS_STYLE}>
+    <FontAwesome icon="spinner" className="fa-fw fa-spin"/>
+    {' '}
+    <em>
+      Getting the latest version...
+    </em>
+  </div>
+);
+
+const FetchingLatestVersionError = () => (
+  <div>
+    <div style={FETCH_STATUS_STYLE}>
+      <FontAwesome icon="times-circle" className="fa-fw" style={{color: 'darkred'}}/>
+      {' '}
+      <strong>
+        There was a problem getting your download link.
+      </strong>
+    </div>
+    <div>
+      Please make sure you are connected to the internet, and
```

Comment:
Why no translations for Maker?

Reply:
:cry: Because no translations for CSD, yet.  I'll go ahead and move all this text into i18n in a followup PR.